### DISCUSSION TITLE
Scoring Theory With Multiple Options

### DISCUSSION THREAD

**Discussion by Cchlee** [krisk/Fuse#716] [Q&A] [answered]
Hi I'm curious about how scoring works when there are results that have a lot more values for the keys than others. For example I'm using Fuse to search on both a name and possible synonyms. The name has a one-to-many relationship with the synonyms. I'm seeing that one of our names, which has many more synonyms than the other names is towards the top of the list every time, no matter what I search on. Updating the key weight does help, but the many synonym name is still towards the top. All search terms, names, and synonyms are relatively short. (<60 characters) I'm curious if I can get a bit more of a breakdown into how the scores work. Are relevance scores aggregated across all keys?

**Comment by krisk** [comment, marked as answer]
Late reply, sorry about that.

Yes, the field norm is likely what's causing this. When a field has many values (e.g. lots of synonyms), the norm calculation takes the total length into account, which can skew scores. You can tune this with the `fieldNormWeight` option (defaults to `1`). Setting it to something lower like `0.5` — or even `0` to disable it entirely — should reduce the bias toward items with more values.
